src/components/pages/about-us/AboutHero.tsx

```diff
@@ -1,10 +1,341 @@
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76H20.5918C20.5854 53.4288 21.901 50.2312 24.2498 47.8689C26.5986 45.5066 29.7886 44.1727 33.1198 44.16C40.1038 44.16 59.9998 53.472 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9518 51.96L52.2478 94.8H67.7518L59.9518 51.96Z"
+              fill="white"
+            />
+          </svg>
+        </div>
+
+        <div className="svg-coconut-tree">
+          <svg
+            width="120"
+            height="120"
+            viewBox="0 0 120 120"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M59.9998 56.76L32.1358 28.8C33.3015 27.6332 34.6858 26.7075 36.2095 26.0759C37.7332 25.4443 39.3664 25.1192 41.0158 25.1192C42.6652 25.1192 44.2984 25.4443 45.8221 26.0759C47.3458 26.7075 48.7301 27.6332 49.8958 28.8C54.8398 33.792 62.3998 54.432 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76L87.8878 28.8C85.5294 26.4419 82.3309 25.1172 78.9958 25.1172C75.6607 25.1172 72.4622 26.4419 70.1038 28.8C65.1598 33.792 57.5998 54.432 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76L32.136 84.6477C30.9655 83.4816 30.0368 82.0958 29.4031 80.5699C28.7694 79.044 28.4432 77.408 28.4432 75.7557C28.4432 74.1035 28.7694 72.4675 29.4031 70.9416C30.0368 69.4157 30.9655 68.0299 32.136 66.8637C37.032 61.9197 57.5998 54.432 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76L87.8638 84.6477C89.0343 83.4816 89.963 82.0958 90.5967 80.5699C91.2304 79.044 91.5566 77.408 91.5566 75.7557C91.5566 74.1035 91.2304 72.4675 90.5967 70.9416C89.963 69.4157 89.0343 68.0299 87.8638 66.8637C82.9678 61.9197 62.3998 54.432 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76H99.4078C99.4142 53.4288 98.0986 50.2312 95.7499 47.8689C93.4011 45.5066 90.211 44.1727 86.8798 44.16C79.8958 44.16 59.9998 53.472 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9998 56.76H20.5918C20.5854 53.4288 21.901 50.2312 24.2498 47.8689C26.5986 45.5066 29.7886 44.1727 33.1198 44.16C40.1038 44.16 59.9998 53.472 59.9998 56.76Z"
+              fill="white"
+            />
+            <path
+              d="M59.9518 51.96L52.2478 94.8H67.7518L59.9518 51.96Z"
+              fill="white"
+            />
+          </svg>
+        </div>
+      </div>
+
+      <div className="absolute bottom-0 w-full">
+        <div className="w-full h-254d bg-gradient-to-b from-transparent to-ruby-red"></div>
+        <div className="w-full h-532d bg-ruby-red"></div>
+      </div>
+
+      <div
+        className="w-full h-screen  absolute top-0 left-0 z-50"
```

Comment:
kalau bisa jangan ada whitespace seperti ini lagi ya gung biar rapi, untuk next aja ketika ada waktu refactor nanti

```suggestion
        className="w-full h-screen absolute top-0 left-0 z-50"
```